src/main/resources/l10n/JabRef_de.properties

```diff
@@ -2323,20 +2323,20 @@ Locked=Gesperrt
 Collect_and_share_telemetry_data_to_help_improve_JabRef.=Sammle_und_teile_Telemetriedaten,_um_bei_der_Verbesserung_von_JabRef_zu_unterstützen.
 Don't_share=Nicht_teilen
 Share_anonymous_statistics=Teile_anonyme_Statistiken
 Telemetry\:_Help_make_JabRef_better=Telemetrie\:_Helfen_Sie,_JabRef_zu_verbessern
-To_improve_the_user_experience,_we_would_like_to_collect_anonymous_statistics_on_the_features_you_use._We_will_only_record_what_features_you_access_and_how_often_you_do_it._We_will_neither_collect_any_personal_data_nor_the_content_of_bibliographic_items._If_you_choose_to_allow_data_collection,_you_can_later_disable_it_via_Options_->_Preferences_->_General.=Zur_Verbesserung_der_Benutzererfahrung_würden_wir_gerne_anonyme_Statistiken_über_die_Features_sammeln,_die_Sie_benutzen._Wir_zeichnen_nur_auf,_welche_Features_Sie_nutzen_und_wie_oft_Sie_diese_benutzen._Es_werden_keinerlei_persönliche_Informationen_über_Sie_oder_den_Inhalt_Ihrer_Bibliografieeinträge_gesammelt._Die_einmal_erlaubte_Datenaufzeichnnung_kann_jederzeit_unter_Optionen->Einstellungen->Allgemein_deaktiviert werden.
-This_file_was_found_automatically._Do_you_want_to_link_it_to_this_entry?=
-Names_are_not_in_the_standard_BibTeX_format.=
-
-Delete_the_selected_file_permanently_from_disk,_or_just_remove_the_file_from_the_entry?_Pressing_Delete_will_delete_the_file_permanently_from_disk.=
-Delete_'%0'=
-Delete_from_disk=
-Remove_from_entry=
-The_group_name_contains_the_keyword_separator_"%0"_and_thus_probably_does_not_work_as_expected.=
-There_exists_already_a_group_with_the_same_name.=
+To_improve_the_user_experience,_we_would_like_to_collect_anonymous_statistics_on_the_features_you_use._We_will_only_record_what_features_you_access_and_how_often_you_do_it._We_will_neither_collect_any_personal_data_nor_the_content_of_bibliographic_items._If_you_choose_to_allow_data_collection,_you_can_later_disable_it_via_Options_->_Preferences_->_General.=Zur_Verbesserung_der_Benutzererfahrung_würden_wir_gerne_anonyme_Statistiken_über_die_Features_sammeln,_die_Sie_benutzen._Wir_zeichnen_nur_auf,_welche_Features_Sie_nutzen_und_wie_oft_Sie_diese_benutzen._Es_werden_keinerlei_persönliche_Informationen_über_Sie_oder_den_Inhalt_Ihrer_Bibliografieeinträge_gesammelt._Die_einmal_erlaubte_Datenaufzeichnnung_kann_jederzeit_unter_Optionen->Einstellungen->Allgemein_deaktiviert_werden.
+This_file_was_found_automatically._Do_you_want_to_link_it_to_this_entry?=Diese_Datei_wurde_automatisch_gefunden._Möchten_Sie_sie_dem_Eintrag_zuordnen?
+Names_are_not_in_the_standard_BibTeX_format.=Namen_entsprechen_nicht_dem_Standard_BibTeX_Format
+
+Delete_the_selected_file_permanently_from_disk,_or_just_remove_the_file_from_the_entry?_Pressing_Delete_will_delete_the_file_permanently_from_disk.=Die_ausgewählte_Datei_unwiderruflich_löschen_oder_nur_vom_Eintrag_entfernen?_Drücken_sie_Entfernen,_um_die_Datei_von_der_Festplatte_zu_löschen.
+Delete_'%0'=Lösche_Datei_'%0'
+Delete_from_disk=Auf_der_Festplatte_löschen
```

Comment:
**Von** der Festplatte?

Reply:
I found "von" and "auf" as both possible when searching for it...